examples/encoder_example.c

```diff
@@ -514,6 +522,15 @@ int main(int argc, char **argv) {
         }
         break;
       }
+      case 'S': {
+        skip = atoi(optarg);
+        if (skip < 1) {
+          fprintf(stderr,
+           "Illegal number of frames to skip (must be a positive integer)\n");
+          exit(1);
```

Comment:
Is there any reason to disallow a skip of 0? It might be convenient for processing from simple scripts, to allow them to unconditionally pass --skip.